view/frontend/web/js/view/payment/method-renderer/alma_monthly_payments.js

```diff
@@ -125,6 +123,22 @@ define(
                 return priceUtils.formatPrice(cents / 100, window.checkoutConfig.priceFormat);
             },
 
+            cartTotal: function () {
+                return priceUtils.formatPrice(window.checkoutConfig.quoteData.grand_total, window.checkoutConfig.priceFormat);
+            },
+
+            customerTotalCostAmount: function (cost) {
+                return priceUtils.formatPrice(cost / 100, window.checkoutConfig.priceFormat);
+            },
+
+            isAnnualInterestRate: function (rate) {
```

Comment:
actually, it seem's to be `hasAnnualInterestRate` no ?

Reply:
For sure

Reply:
Fixed